dcbr-web/src/components/MenuOperationLocation.vue

```diff
@@ -79,20 +79,13 @@ export default {
     hasAdditionalLocations: {
       // getter
       get() {
-        if (this.visited == false) {
-          return "";
-        } else {
-          return this.$store.getters["operationLocations/hasAdditionalLocations"];
-        }
+        return this.$store.getters["operationLocations/hasAdditionalLocations"];
       },
       // setter
       set(value) {
-        if (this.hasAdditionalLocations == "") {
-          this.visited = true;
-        }
         this.$store.dispatch("operationLocations/hasAdditionalLocations", value);
-        if(value == true) {
-          if(value == true && this.$store.getters["operationLocations/locations"].length == 0) {
+        if(value == "true") {
```

Comment:
Instead of nesting the `if ` statements, you could just use an `AND`, it makes the code cleaner to read: 
```
if(value == true && ["operationLocations/locations"].length == 0)){
```

It is also best practice to use [strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#Strict_equality_using) when comparing values, as it will check for both type and value and

Reply:
Pushed change to my fork.

Reply:
Thanks! Please take a look at the strict equality operators as well, we should be using `===` instead of `==` (and negated variants as well).

Reply:
Done! Ready for review